mkdocs.yml

```diff
@@ -61,3 +73,13 @@ extra_css:
 plugins:
   - search
   - include-markdown
+  - mkdocstrings:
+      handlers:
+        python:  # configuration options are documented at https://mkdocstrings.github.io/python/usage/#configuration
+          paths: [src/dsp_tools/]
+          options:
+            members_order: source
+            show_labels: false
+            show_signature: false
+            show_docstring_raises: false
+            show_root_toc_entry: false
```

Comment:
(According to Noémi, the "Raises" section in the docstring is not helpful. They must be present in the code (because of darglint), so I decided to mute them here.

Reply:
I deleted the warnings in the code, but I cannot delete the raises (because of darglint). For this reason, I have to disable the raises here, but not the warnings.